### PR TITLE
LHC-125 update mega menu to be click activated and apply redesign styling

### PR DESCRIPTION
**Pull request by rdai10**
### Description
https://issues.liferay.com/browse/LHC-125

### Checklist
- [x] Code compiles without errors.
- [ ] Include tests for new features and functionality.
- [ ] Update README/documentation, when applicable.
- [ ] All tests, new and existing, pass without errors.
- [ ] Checked browser compatibility, especially IE11.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,14 @@
 
 This repo contains a large file uploader app and a theme for Liferay's Zendesk Help Center.
 
+## Large File Uploader App
+
+A simple app that shows up in the agent view that displays a link to an external large file uploader page that allows the user to upload a file greater than the Zendesk attachment file size limit.
```
Comment (jonmak08) [inline README.md]: Can we update this text to `A simple app that appears in the agent view; it displays a link to a page with an external large file uploader allowing users to upload a file larger than the Zendesk attachment file size limit.`

File: README.md [hunk 2]
```diff
@@ -2,7 +2,14 @@
...
+
 ## Theme
+
+The Guide theme for Liferay's Help center.
```
Comment (jonmak08) [inline README.md]: Capitalize `Center` to keep it consistent with other references to `Liferay's Help Center`.

Comment (jonmak08) [reply README.md]: Should this also say `Liferay's Zendesk Help Center` to be consistent with everything?

File: README.md (hunk 1 above)
Comment (rdai10) [reply README.md]: :ok_hand:

File: README.md (hunk 2 above)
Comment (rdai10) [reply README.md]: :ok_hand:

**Pull request merged**